Pod/Classes/APMAlertController.swift

```diff
@@ -28,7 +30,7 @@ import SnapKit
     fileprivate var topScrollViewHeightConstraint: Constraint?
     fileprivate let contentView = UIView()
     fileprivate var anyTitleObject: AnyObject
-    fileprivate var topTitleMessageSeparatorConstraint: Constraint?
+    fileprivate var titleMessageSeparatorConstraint: Constraint?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/almas-dev/APMAlertController/blob/b9c4a952162e21ef3d6579eda097becf88e6343a/Pod/Classes/APMAlertController.swift#L33"></span>Variable name should be between 2 and 24 characters long: ‘titleMessageSeparatorConstraint’</td>
    </tr>
  </tbody>
</table>